src/animatediff/generate.py

```diff
@@ -1123,12 +1126,30 @@ def run_inference(
 ):
     out_dir = Path(out_dir)  # ensure out_dir is a Path
 
+    # Trim and clean up the prompt for filename use
+    prompt_map = region_condi_list[0]["prompt_map"]
+    prompt_tags = [re_clean_prompt.sub("", tag).strip().replace(" ", "-") for tag in prompt_map[list(prompt_map.keys())[0]].split(",")]
+    prompt_str = "_".join((prompt_tags[:6]))[:100]
```

Comment:
There was a pattern of exceeding the maximum path length for windows, so I'm limiting it to 50.

Reply:
ahh I just looked it up and apparently windows' max length is for the whole path, not just the filename like linux. then the limit of 50 makes more sense